client/my-sites/email/google-workspace-add-users/index.jsx

```diff
@@ -52,7 +52,7 @@ import { getProductsList } from 'calypso/state/products-list/selectors/get-produ
  */
 import './style.scss';
 
-class GSuiteAddUsers extends React.Component {
+class GoogleWorkspaceAddUsers extends React.Component {
```

Comment:
These components are still in use for existing G Suite customers, so I am not sure that a refactor/rename makes sense yet. It may also be better to push that into a separate PR.